capsules/extra/src/atecc508a.rs

```diff
@@ -0,0 +1,836 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2023.
+
+//! Capsule for interfacing with the ATECC508A CryptoAuthentication Device
+//! using the I2C bus.
+//!
+//! <https://cdn.sparkfun.com/assets/learn_tutorials/1/0/0/3/Microchip_ATECC508A_Datasheet.pdf>
```

Comment:
Perhaps a more official link:

```suggestion
//! <https://ww1.microchip.com/downloads/en/DeviceDoc/20005928A.pdf>
```